python/lsst/rucio/register/rucio_interface.py

```diff
@@ -359,3 +378,20 @@ def register_zips(self, dataset_id, zip_files) -> int:
         self._add_replicas(bundles)
         self.register_to_dataset(bundles)
         return len(bundles)
+
+    def register_dims(self, dataset_id, dim_files) -> int:
```

Comment:
As you included a type hint for the return value, could you please add the type hints to the function arguments as well?

Reply:
Added